skimage/feature/peak.py

```diff
@@ -24,17 +24,16 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
     min_distance : int, optional
         Minimum number of pixels separating peaks in a region of `2 *
         min_distance + 1` (i.e. peaks are separated by at least
-        `min_distance`). If `exclude_border` is True, this value also excludes
-        a border `min_distance` from the image boundary.
+        `min_distance`).
         To find the maximum number of peaks, use `min_distance=1`.
     threshold_abs : float, optional
         Minimum intensity of peaks. By default, the absolute threshold is
         the minimum intensity of the image.
     threshold_rel : float, optional
         Minimum intensity of peaks, calculated as `max(image) * threshold_rel`.
-    exclude_border : bool, optional
-        If True, `min_distance` excludes peaks from the border of the image as
-        well as from each other.
+    exclude_border : int, optional
+        If nonzero, `exclude_border` excludes peaks from
+        within `exclude_border` of the border of the image.
```

Comment:
Could you reformulate this?
Like, for example, `peaks within 'exclude_border'-pixels frame will be excluded`.